benchmarks/benchmark.py

```diff
@@ -24,236 +24,213 @@
 csv_writer = csv.writer(csv_file)
 csv_writer_timed = csv.writer(csv_file_timed)
```

Comment:
This is something from a previous PR I missed, but `csv_file` is just named "results.csv". Should this instead be something more useful like the timed one (which is "/output/DATE_TIME.csv")? (I'm not 100% sure what the difference between the two csvs is)

Reply:
Oh! I think the two output files are the same: I did a diff and they are identical!